conf/cassandra_latest.yaml

```diff
@@ -180,37 +184,57 @@ authenticator:
 # Authorization backend, implementing IAuthorizer; used to limit access/provide permissions
 # Out of the box, Cassandra provides org.apache.cassandra.auth.{AllowAllAuthorizer,
 # CassandraAuthorizer}.
+# Optional parameters can be specified in the form of:
+#   parameters:
+#    param_key1: param_value1
+#    ...
 #
 # - AllowAllAuthorizer allows any action to any user - set it to disable authorization.
 # - CassandraAuthorizer stores permissions in system_auth.role_permissions table. Please
 #   increase system_auth keyspace replication factor if you use this authorizer.
-authorizer: AllowAllAuthorizer
+authorizer:
+  class_name: org.apache.cassandra.auth.AllowAllAuthorizer
```

Comment:
all these changes in cassandra_latest.yaml  can also be applied to cassandra.yaml (this being the trunk patch)

Reply:
done.

Reply:
No worries. Yes, code changes allow both FQCN and short form and unfortunately they were using a mix of both. I'm fine using either way, and since the FQCN seem to be the predominant one I've changed it that. Should we standardize on both `cassandra.yaml` and `cassandra_latest.yaml` for all 5.0 and trunk on the usage of the short form? If not, please clarify which forms should be used where.

Reply:
it's easy to be wrong about these things, but my understanding is the precedence is to use short names for actual values, and also FQNs in the comments so that it's intuitive that both can be used.

Reply:
to clarify, in this PR and the 5.0 PR, on lines where `class_name: ` is introduced then use short class names.

ignore lines that are not otherwise being touched in these PRs.

Reply:
Agreed to use short class names in all places and fixing other places not following that convention to prevent confusion in the future.